inlong-agent/agent-core/src/main/java/org/apache/inlong/agent/core/conf/ConfigJetty.java

```diff
@@ -62,10 +62,11 @@ public ConfigJetty(JobManager jobManager, TriggerManager triggerManager) {
     }
 
     private void initJetty() throws Exception {
-        ServerConnector connector = new ServerConnector(this.server);
-        connector.setPort(conf.getInt(
-                AgentConstants.AGENT_HTTP_PORT, AgentConstants.DEFAULT_AGENT_HTTP_PORT));
-        server.setConnectors(new Connector[]{connector});
+        try (ServerConnector connector = new ServerConnector(this.server)) {
+            connector.setPort(conf.getInt(
+                    AgentConstants.AGENT_HTTP_PORT, AgentConstants.DEFAULT_AGENT_HTTP_PORT));
+            server.setConnectors(new Connector[]{connector});
+        }
```

Comment:
It seems that the server needs to hold this connector rather than close it immediately.
The connector will be closed when call server.stop().

Reply:
restored